<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>331</width>
      <height>376</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QTextEdit" name="age_text"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="gender_button">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="name_text"/>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="name_button">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="motivation_text"/>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="gender_text"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="age_button">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="motivation_button">
       <property name="text">
        <string>Motivation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="race_button">
       <property name="text">
        <string>Race</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="race_text"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
